Guide/realtime-spas.markdown

```diff
@@ -686,8 +686,59 @@ DELETE FROM todos WHERE id = "66cc037e-5729-435c-b507-a17492fe44f4"
 ```
 
 
+## Using DataSync library with pure JavaScript
```

Comment:
I think `pure` could be confusing here, maybe `without a bundler` is more precise?

```suggestion
## Using DataSync library without a bundler
```